src/modules/Dashboard/features/RespondentData/RespondentDataSummary/RespondentDataSummary.utils.tsx

```diff
@@ -430,6 +464,43 @@ export const formatActivityItemAnswers = (
         ],
       };
     }
+    case ItemResponseType.SingleSelectionPerRow:
```

Comment:
Please add unit tests to RespondentDataSummary.utils.test.tsx to cover this case.